package.json

```diff
@@ -31,7 +35,11 @@
     "test:coverage": "react-scripts test --watchAll=false --coverage",
     "test:watch": "react-scripts test --watchAll",
     "lint": "eslint ./src/**/*.ts ./src/**/*.tsx",
-    "lint:fix": "eslint --fix ./src/**/*.ts ./src/**/*.tsx"
+    "lint:fix": "eslint --fix ./src/**/*.ts ./src/**/*.tsx",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
```

Comment:
I don't think we want to add these back - could you remove them please?

Reply:
Have removed eslinConfig but it isn't working without "eject"
Can we keep it there?

Reply:
we don't need eject - do you just have a trailing comma on the previous line or something?